delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/RawResult.scala

```diff
@@ -0,0 +1,49 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.client
+
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.BNode
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError.ConversionError
+import ch.epfl.bluebrain.nexus.delta.rdf.graph.{Dot, Graph, NTriples}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.api.{JsonLdApi, JsonLdOptions}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.{ContextValue, RemoteContextResolution}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.{CompactedJsonLd, ExpandedJsonLd}
+import io.circe.JsonObject
+import monix.bio.IO
+
+/**
+  * An elasticsearch raw query result
+  */
+final case class RawResult(value: JsonObject)
+
+object RawResult {
+  implicit val queryResultJsonLdEncoder: JsonLdEncoder[RawResult] = new JsonLdEncoder[RawResult] {
+
+    override def context(value: RawResult): ContextValue = ContextValue.empty
+
+    override def expand(
+        value: RawResult
+    )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, ExpandedJsonLd] =
+      IO.raiseError(ConversionError("An ElasticSearch response cannot be converted to expanded Json-LD", "expanding"))
+
+    override def dot(
+        value: RawResult
+    )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, Dot] =
+      IO.raiseError(ConversionError("An ElasticSearch response cannot be converted to Dot", "to dot"))
+
+    override def ntriples(
+        value: RawResult
+    )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, NTriples] =
+      IO.raiseError(ConversionError("An ElasticSearch response cannot be converted to N-Triples", "to ntriples"))
+
+    override def graph(
+        value: RawResult
+    )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, Graph] =
+      IO.raiseError(ConversionError("An ElasticSearch response cannot be converted to Graph", "to graph"))
+
+    override def compact(
+        value: RawResult
+    )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, CompactedJsonLd] =
+      IO.pure(CompactedJsonLd.unsafe(BNode.random, ContextValue.empty, value.value))
```

Comment:
ES query results are not JSON-LD and I don't think we should treat them as such. I don't really see the need for this `JsonLdEncoder`.

Reply:
This `JsonLdEncoder` odes exactly that, return pure json without any modifications. And as you can see it won't work with expanded nor other weird things.

Why is there? Because it is much more convenient to keep using the current `emit` which will deal with the error channel and the content negotiation without having to code again for it

Reply:
Won't it return an exception when someone will try to request expanded JSON-LD or other of those formats and interfere with regular content negotiation?

Reply:
If you try to use this endpoint having Accept: `application/n-triples` it will trigger the exception, yes. However this exception says quite clearly that `An ElasticSearch response cannot be converted to N-Triples` so I don't find it wrong.

If you thing that's bad I can add some special case to handle `Json` on the `emit`. But it seems an unnecessary overhead to me

Reply:
We can do it this way, but we just have to make sure that we return `406 Not Acceptable` when someone requests n-triples etc.

Reply:
I've handled it the proper way, thought the `DeltaDirectives`